preview_generator/preview/builder/image__pillow.py

```diff
@@ -167,18 +167,18 @@ def __init__(self, logger: logging.Logger):
         assert logger is not None
         self.logger = logger
 
-    def get_strategy(self, mode: str) -> ImageConvertStrategy:
+    def get_strategy(self, image: PIL.Image) -> ImageConvertStrategy:
         """
         Get strategy to use for this image mode
         :param mode: pillow image mode
         :return:
         """
-        if mode == 'RGBA':
-            return RGBAImageConvertStrategy(self.logger)
-        elif mode == 'LA':
-            return LAImageConvertStrategy(self.logger)
+        if image.mode == 'RGBA':
+            return ImageConvertStrategyRGBA(self.logger)
+        elif image.mode == 'LA':
+            return ImageConvertStrategyLA(self.logger)
         else:
-            return NotTransparentImageConvertStrategy(self.logger)
+            return ImageConvertStrategyNOTTRANSPARENT(self.logger)
```

Comment:
last suggestion: Not transparent strategy is not concrete... maybe do not use a uppercase naming in this case? (this is really a suggestion/question - if you feel it's not clear, tell it and we'll skip this comment)

Reply:
yes, we should do same for transparent one too.